application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -4291,6 +4296,22 @@ input PlanDateTimeInput @oneOf {
   latestArrival: OffsetDateTime
 }
 
+"A collection of selectors for what routes/agencies should be included in or excluded from the search."
+input PlanFilterInput {
+  "A list of selectors for what routes/agencies should be excluded during the search. If route/agency/mode matches with at least one selector it will be excluded. An empty list or a list containing `null` is forbidden."
+  not: [PlanFilterSelectInput!]
+  "A list of selectors for what routes/agencies should be allowed during search. In order to be accepted a route/agency has to match with at least one selector. An empty list or a list containing `null` is forbidden."
+  select: [PlanFilterSelectInput!]
```

Comment:
Consider using "exclude/include" instead of "not/select" - I do not have an opinion, and I have not check if there are semantic differences.